### ISSUE TITLE
Prevent default description tag to be inserted into HTML if not updated

### ISSUE THREAD

**Issue by kuanb** [bjornbjorn/SEO-Lite#35]
I don't use this tool or Expression Engine CMS but wanted to note that I was c/p a link into Slack today from these guys: http://mrcagney.com/

And the description was bizarre:
```
<meta name='keywords' content='your, default, keywords, here' />
<meta name='description' content='Your default description here' />
```

Inspecting the HTML, I saw the tag that noted the above was created b/c of this SEO Lite add on. Perhaps it would be ideal if the add on did not insert these meta tags if the user has not updated the values associated with each.

**Comment by bjornbjorn** [comment]
Hi kuanb,

thanks for your input. 

These are the default values you get when installing SEO Lite. Most people will change those values on install, but if they don't it will look like what you pasted. I guess a solution could be to have empty values on install, that way it would look like this:

```
<meta name='keywords' content='' />
<meta name='description' content='' />
```

.. but then I might get more support requests asking why keywords / descriptions are empty (on the frontpage etc). The way they are today they sort of push you to go and investigate and then discover the settings in the backend.

Or maybe just write it out like this:

```
<meta name='keywords' content='Please edit the default keywords setting in SEO Lite' />
<meta name='description' content='Please edit the default description setting in SEO Lite' />
```
🤔

**Comment by kuanb** [comment]
Yeah sorry I can't be of more help. I'm not sure what your plug in does. If it's a GUI over html perhaps dynamically converting those segments to empty quotes during a build step would work?

Downside is the more obvious you make the default text in the current set up, the more ridiculous it will be when someone doesn't update it!
